### ISSUE TITLE
Julia can not be started in VS code

### ISSUE THREAD

**Issue by CaoHua1203** [julia-vscode/julia-vscode#2462]
Dears,

Julia can not be started in VS Code after the julia extension was updated.

There is an error saying the **julia executable path** is not correct. However, it is actually right.

The issue occured under new extension version 1.4.3 and 1.4.0.
When I retried the older version like 1.3.34, there is no such error and Julia can be started.

May I have your help to solve the issue?

Thanks & B.Rgds.

**Comment by darsnack** [comment]
I encountered this error after the release of the persistent tmux sessions feature. After enabling (and then disabling) the feature, I could no longer launch a new REPL with the same error as above. This is all on a remote session (it works fine on my local machine).

**Comment by pfitzseb** [comment]
Shell variables should never have been respected, Julia was always only started directly without involving a shell.

**Comment by darsnack** [comment]
I am not sure how it worked on my remote machine in the past then. I've always manually unpacked the Julia download from the website into `/opt`. The only place I made the machine aware of the install was in my `~/.zshrc`.

**Comment by CaoHua1203** [comment]
Tried the latest extension v.1.5.4, while the issue still exists.
Uninstalled Julia and VS Code, then re-installed them, but the issue can not be solved. :(

![VS Code Extension Issue](https://user-images.githubusercontent.com/19659749/141715591-ecd46292-aeb3-4f82-8c98-f3eb0703427e.png)![VS Code Extension Issue](https://user-images.githubusercontent.com/19659749/141715591-ecd46292-aeb3-4f82-8c98-f3eb0703427e.png)

**Comment by CaoHua1203** [comment]
The issue was solved. Upon comparing the difference between 1.3.34 and 1.4.0, I found it was caused due to some modification since version 1.4.0.

Open “.vscode\extensions\julialang.language-julia-1.5.6\package.json” (as below screenshot)，change ”configuration”–>”julia.executablePath”–>”scope” from ”machine-overridable” into ”machine”, i.e. the same as in version 1.3.34.

![VS Code Julia Extension Modification](https://user-images.githubusercontent.com/19659749/143162812-93de7758-1950-4ad1-8ec3-617538e98ba5.png)

Restart VS Code, then Julia can be started.

While in Julia Discourse, @pfitzseb reminded me that the issue might be caused due to incorrect workspace configuration.

I am not familiar with workspace and never configurated or changed it previously. So I did not realize that is the reason causing the issue.

Then I checked the configuration of workspace and found it is wrong.

After amendment of the workspace configuration, the issue was solved.

Thanks to @pfitzseb for his warm help. :)
